internal/cluster/view.go

```diff
@@ -0,0 +1,46 @@
+package cluster
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/lipgloss/table"
+)
+
+func ClusterSummary(ctx context.Context, OrganizationSlug string, ClusterID string, f *factory.Factory) (string, error) {
+
+	clusterSummary, err := QueryCluster(ctx, OrganizationSlug, ClusterID, f)
```

Comment:
It's not critical now but I think this would be better as separate functions. One that retrieves cluster information from the API and one that takes that and renders a string - instead of in the same function. That would lead to easier testing as well

Reply:
Sure Jarryd! I can take this up as next iteration